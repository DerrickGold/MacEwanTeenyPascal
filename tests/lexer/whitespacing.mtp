if identifier == 'string' and id2 == 567