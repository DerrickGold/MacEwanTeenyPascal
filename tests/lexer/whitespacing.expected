1, TOK_KEY_IF, if
1, TOK_ID, identifier
1, TOK_EQ, =
1, TOK_EQ, =
1, TOK_STR, 'string'
1, TOK_KEY_AND, and
1, TOK_ID, id2
1, TOK_EQ, =
1, TOK_EQ, =
1, TOK_NUM, 567
1, TOK_ENDFILE,
